composer.json

```diff
@@ -24,6 +24,7 @@
         "league/fractal": "^0.16.0"
     },
     "require-dev": {
+        "php": "^7.0",
```

Comment:
After looking at the code and unit tests it appears 7.1 is going to be required which means the docs should be updated. The issue here is that adoption into older versions of laravel/lumen may be more difficult as they often don't have such high php version requirements.